Description: Fix offset data type in examples.
Author: Martin Pitt <martin.pitt@ubuntu.com>
Bug: http://code.google.com/p/libarchive/issues/detail?id=282
Applied-Upstream: b8bb251083006b8af56c4d099ac4dc5475ce5cbb, 7da8d576848982f2b0264febf89cb8e3f3580af2

--- a/examples/minitar/minitar.c
+++ b/examples/minitar/minitar.c
@@ -400,7 +400,7 @@
 	int r;
 	const void *buff;
 	size_t size;
-	off_t offset;
+	int64_t offset;
 
 	for (;;) {
 		r = archive_read_data_block(ar, &buff, &size, &offset);
--- a/examples/untar.c
+++ b/examples/untar.c
@@ -199,7 +199,7 @@
 	int r;
 	const void *buff;
 	size_t size;
-#if ARCHIVE_VERSION >= 3000000
+#if ARCHIVE_VERSION_NUMBER >= 3000000
 	int64_t offset;
 #else
 	off_t offset;
